### DISCUSSION TITLE
Using Japanese Mozc in hyprland

### DISCUSSION THREAD

**Discussion by luckasRanarison** [hyprwm/Hyprland#2838] [Q&A] [answered]
Hey everyone! It's my first time trying a WM and hyprland is just amazing but I've been wondering how can I use japanese input in Hyprland?

**Comment by vaxerski** [comment, marked as answer]
use fcitx5 and autostart it with exec once

**Reply by luckasRanarison** [reply]
Thanks for the help and thanks for your amazing work, I love hyprland
